go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -328,6 +328,9 @@ type folderBranchOps struct {
 
 	// Closed on shutdown
 	shutdownChan chan struct{}
+	// Closed once we're done shutting down. Any goroutine that logs must be
```

Comment:
By searching around in this file for `go `, I see a number of goroutines that you aren't tracking with this waitgroup.  I guess some of them are because they're already being tracked by another waitgroup, like `fbo.partialSyncs` and `fbo.editActivity`.  However those are only waited on as part of `SyncFromServer`, which is only called during shutdown by the first `KBFSOps` instance to shutdown, so those goroutines could still cause problems for the other instances.

There are others, like the one in `fbo.commitHeadLocked()` that only log in error conditions -- maybe we don't care about those ones?  Though they could still easily get shutdown errors, and thus log.

Reply:
Yeah I only tracked long running `go fbo.*` goroutines. Do you think I should also track `go func()` ones?